packages/react/src/themes/teams/components/Label/labelVariables.ts

```diff
@@ -9,29 +9,44 @@ export interface LabelVariables {
   startPaddingLeft: string
   endPaddingRight: string
   height: string
-  iconColor: string
+}
+
+export const invertColorScheme = colorScheme => {
+  return Object.keys(colorScheme).reduce((accumulator, key) => {
+    const foreground = colorScheme[key].foreground
+    const background = colorScheme[key].background
+    return {
+      ...accumulator,
+      [key]: {
+        ...colorScheme[key],
+        foreground: background,
+        background: foreground,
+      },
+    }
+  }, {})
 }
 
 export default (siteVars: SiteVariablesPrepared): LabelVariables => {
-  const color = 'rgba(0, 0, 0, 0.6)'
+  const invertedColorScheme = invertColorScheme(siteVars.colorScheme)
 
   return {
-    colorScheme: extendColorScheme(siteVars.colorScheme, {
+    colorScheme: extendColorScheme(invertedColorScheme, {
+      // TODO use here some color scheme values, after the designs are delivered
       default: {
-        background: color,
-        foreground: 'rgb(232, 232, 232)',
+        foreground: 'rgba(0, 0, 0, 0.6)',
```

Comment:
Would really like to use here some color scheme values, because these overrides now goes to all themes (this is why we have to add labelVariables in teams and HC theme), which are actually partially copies from the light theme :(